External/FEXCore/Source/Interface/IR/Passes/RegisterAllocationPass.cpp

```diff
@@ -254,15 +255,30 @@ namespace {
   }
 
   void FreeRegisterGraph(RegisterGraph *Graph) {
+    if (Graph->Nodes) {
+      ::munmap(Graph->Nodes, Graph->NodeCount * sizeof(RegisterNode));
+    }
     delete Graph;
   }
 
   void ResetRegisterGraph(RegisterGraph *Graph, uint64_t NodeCount) {
     NodeCount = AlignUp(NodeCount, sizeof(uint64_t));
-    Graph->Nodes.clear();
-    Graph->Nodes.resize(NodeCount);
+    if (Graph->NodeCount < NodeCount) {
```

Comment:
Graph->NodeCount really should be aligned up to the number of nodes that can fit on a page. 

Might prevent a few unnecessary resizes during startup.

Reply:
Good idea. It now rounds up to how many nodes can fit in to a page.